src/nnet3bin/nnet3-get-egs.cc

```diff
@@ -38,10 +38,12 @@ static bool ProcessFile(const GeneralMatrix &feats,
                         bool compress,
                         int32 num_pdfs,
                         UtteranceSplitter *utt_splitter,
-                        NnetExampleWriter *example_writer) {
+                        NnetExampleWriter *example_writer,
+                        int32 length_tolerance = 2) {
```

Comment:
since length_tolerance is an input, put it before outputs.  No need to preserve old interface as this function is internal to the file.